kogitoq/kogito-common/src/main/java/life/genny/kogito/common/core/Dispatch.java

```diff
@@ -393,9 +395,18 @@ public void handleDropdownAttributes(Ask ask, String parentCode, BaseEntity targ
 		// check for dropdown attribute
 		if (ask.getQuestion().getAttribute().getCode().startsWith(Prefix.LNK_)) {
 
-			// get list of value codes
-			List<String> codes = beUtils.getBaseEntityCodeArrayFromLinkAttribute(target,
-					ask.getQuestion().getAttributeCode());
+			List<String> codes = null;
```

Comment:
set to new ArrayList<>() here. SONARLint compliant + on line 411 means we only have to check if codes.isEmpty (code cleanliness)

Reply:
Done